build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+pkgname=$1
+cur_path=$(cd "$(dirname "$0")"; pwd)
+if [ "$pkgname" = "ipmitool" ]; then
```

Comment:
My suggest is, check all the subdirectory in this repo, if there is build.sh, run it.
So, don't need to check whether to build `ipmitool`
And, you need to replace ipmitool with veriable `pkgname` in line 6